src/implementations/truncation.jl

```diff
@@ -28,82 +28,93 @@ end
 
 # findtruncated
 # -------------
+# Generic fallback
+function findtruncated_svd(values::AbstractVector, strategy::TruncationStrategy)
+    return findtruncated(values, strategy)
+end
+
 # specific implementations for finding truncated values
 findtruncated(values::AbstractVector, ::NoTruncation) = Colon()
 
-function findtruncated(values::AbstractVector, strategy::TruncationKeepSorted)
-    howmany = min(strategy.howmany, length(values))
-    return partialsortperm(values, 1:howmany; by=strategy.by, rev=strategy.rev)
-end
-function findtruncated_sorted(values::AbstractVector, strategy::TruncationKeepSorted)
+function findtruncated(values::AbstractVector, strategy::TruncationByOrder)
     howmany = min(strategy.howmany, length(values))
-    return 1:howmany
+    return partialsortperm(values, 1:howmany; strategy.by, strategy.rev)
+end
+function findtruncated_svd(values::AbstractVector, strategy::TruncationByOrder)
+    if strategy.by === abs
+        howmany = min(strategy.howmany, length(values))
+        return strategy.rev ? (1:howmany) : ((length(values) - howmany + 1):length(values))
+    else
+        return findtruncated(values, strategy)
+    end
 end
```

Comment:
```suggestion
function findtruncated_svd(values::AbstractVector, strategy::TruncationByOrder)
    strategy.by === abs || return findtruncated(values, strategy)
    howmany = min(strategy.howmany, length(values))
    return strategy.rev ? (1:howmany) : ((length(values) - howmany + 1):length(values))
end
```
Just to match the style of `findtruncated_svd(values::AbstractVector, strategy::TruncationByValue)` (also personally I like that style better).